plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestClickHouseLatestConnectorTest.java

```diff
@@ -35,4 +37,11 @@ protected QueryRunner createQueryRunner()
                         .buildOrThrow(),
                 REQUIRED_TPCH_TABLES);
     }
+
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        // The numeric value depends on file system
+        return OptionalInt.of(255 - ".sql.detached".length());
```

Comment:
where comes `.sql.detached` from?

Reply:
It comes from ClickHouse internal implementation.